Hunting Queries/OfficeActivity/double_file_ext_exes.yaml

```diff
@@ -0,0 +1,36 @@
+id: d12580c2-1474-4125-a8a3-553f50d91215
+name: Exes with double file extentsion and access summary
+description: |
+  'Provides a summary of executable files with double file extensions in SharePoint 
+   and the users and IP addresses that have accessed them.'
+requiredDataConnectors:
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity
+tactics:
+  - Defense Evasion
+relevantTechniques:
+  - T1036
+query: |
+
+  let timeframe = 14d;
+  let known_ext=dynamic(["lnk","log","option","config", "manifest", "partial"]);
+  OfficeActivity
+  | where TimeGenerated > ago(timeframe)
+  | where RecordType =~ "SharePointFileOperation"
+  | where SourceFileExtension !in~ (known_ext)
+  | where OfficeObjectId contains ".exe."
+  | extend Extension = extract("[^.]*.[^.]*$",0, OfficeObjectId)
+  | join kind= leftouter ( 
+    OfficeActivity
+      | where TimeGenerated > ago(timeframe)
+      | where RecordType =~ "SharePointFileOperation"
+      | where Operation =~ "FileDownloaded" or Operation =~ "FileAccessed" 
```

Comment:
Might be more efficient if we use this with the above where clause via an "and" since we are working across 14d of data.